src/bin/tikv-ctl.rs

```diff
@@ -415,6 +415,28 @@ trait DebugExecutor {
 
     fn check_local_mode(&self);
 
+    fn verify_all_regions(&self, mgr: Arc<SecurityManager>, cfg: &PdConfig, region_ids: Vec<u64>) {
```

Comment:
I think "verify_regions" is more appropriate since it only verifies the specified regions.